### PR TITLE
[fix](regression-test) line_delimiter parse error in regression_test test_tvf_based_broker_load

### PR DESCRIPTION
**Pull request by 133tosakarin**
add compress_type for s3 tvf,  and the broker load is async, so we should wait  loading finished
fix some test files content error

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by 133tosakarin** [comment]
run buildall

**Comment by 133tosakarin** [comment]
run buildall

File: regression-test/suites/load_p2/broker_load/test_tvf_based_broker_load.groovy [hunk 1]
```diff
@@ -225,12 +304,20 @@ suite("test_tvf_based_broker_load_p2", "p2") {
                 }
             }
 
+
+            logger.info("test parquet_s3_case1")
             order_qt_parquet_s3_case1 """select count(*) from parquet_s3_case1 where col1=10"""
+            logger.info("test parquet_s3_case2")
```
Comment (HHoflittlefish777) [inline regression-test/suites/load_p2/broker_load/test_tvf_based_broker_load.groovy:310]: log info still need?

**Comment by liaoxin01** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by sollhui — APPROVED** [review]
LGTM

**Pull request merged**